js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+            'lastTradeTimestamp': undefined,
+            'symbol': order['market'],
+            'type': order['ord_type'],
+            'side': order['side'],
+            'price': this.safeFloat (order, 'price'),
+            'average': this.safeFloat (order, 'avg_price'),
+            'amount': this.safeFloat (order, 'volume'),
+            'remaining': this.safeFloat (order, 'remaining_volume'),
+            'filled': this.safeFloat (order, 'executed_volume'),
+            'status': order['state'],
+            'fee': undefined,
+        };
+        return result;
+    }
+
+    async createMultiOrders (params = {}) {
+        let response = await this.privatePostOrdersMulti (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    async filterOrders (params = {}) {
+        let response = await this.privateGetOrdersFilter (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        let response = await this.privatePostOrderDelete (this.extend (params));
+        const data = response['data'];
+        return this.parseOrder (data);
+    }
+
+    async cancelMulitOrders (params = {}) {
+        let response = await this.privatePostOrderDeleteMulti (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    async cancelAllOrders (symbol = undefined, params = {}) {
+        let response = await this.privatePostOrdersClear (this.extend (params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    sign (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
+        let url = this.urls['api'] + this.implodeParams (path, params);
+        let query = this.omit (params, this.extractParams (path));
+        if (Object.keys (query).length) {
+            url += '?' + this.urlencode (query);
+        }
+        headers = { 'Content-Type': 'application/json' };
+        return { 'url': url, 'method': method, 'body': body, 'headers': headers };
+    }
+
+    handleErrors (code, reason, url, method, headers, body, response) {
+        // API response structure
+        // HTTP code : 200 (success to send request to API) / 201 (success to create order)
+        // {"code": , (success return 0. other code indicate error)
+        //  "message": "", (if error, contain error message report)
+        //  "data": []}
+        if (response === undefined) {
+            return;
+        }
+        if (code === 200 || code === 201) {
+            const resCode = this.safeValue (response, 'code');
+            if (resCode === undefined) {
+                throw new ExchangeError (response);
+            }
+            if (resCode !== 0) {
+                throw new ExchangeError (response['message']);
+            }
+        }
+    }
+};
```

Comment:
Need to add a newline here to make it pass the syntax checks:
https://travis-ci.org/ccxt/ccxt/builds/530571796#L1465